.ci/release.groovy

```diff
@@ -67,24 +71,28 @@ pipeline {
 }
 
 def getVaultSecretRetry(Map args){
-  def secret = arg.containsKey('secret') ? args.secret : error('Secret not valid')
+  def secret = args.containsKey('secret') ? args.secret : error('Secret not valid')
```

Comment:
One more type found? :-)

Reply:
It is the same, this is probably because I did not rebase the changes on master, I will do